crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1930,6 +1930,10 @@ impl SymbolicationActor {
         let timer_old = Instant::now();
         let process_state = ProcessState::from_minidump(&minidump, Some(&cfi))?;
         let timer_old = timer_old.elapsed();
+
+        metric!(timer("minidump.stackwalk.duration") = timer_old, "method" => "old");
+        metric!(counter("minidump.stackwalk.results") += 1, "equality" => "not tested");
```

Comment:
might want to put this behind an `if !compare_stackwalking_methods` (or use an else branch on the existing if).  Otherwise you'll start double counting if it's enabled